tests/rptest/services/action_injector.py

```diff
@@ -224,10 +243,21 @@ def create_context_with_defaults(redpanda: RedpandaService,
                                  *args,
                                  **kwargs) -> ActionCtx:
     admin = Admin(redpanda)
+
+    if op_type.is_reversible:
+        min_time_between_actions = max(int(1 / redpanda.num_nodes * 60), 10)
+        max_time_between_actions = min_time_between_actions * 2
+    else:
+        min_time_between_actions = 10
+        max_time_between_actions = 30
+    redpanda.logger.info(
+        f'Initializing ActionCtx with '
+        f'min time between actions: {min_time_between_actions}, '
+        f'max time between actions: {max_time_between_actions}')
     config = config or ActionConfig(
         cluster_start_lead_time_sec=20,
-        min_time_between_actions_sec=10,
-        max_time_between_actions_sec=30,
+        min_time_between_actions_sec=min_time_between_actions,
+        max_time_between_actions_sec=max_time_between_actions,
```

Comment:
Looks like this change and the priority queue change are independent. If that is the case, they should be in separate commits.

Reply:
I have removed this change now, initially I thought it would help with the issue but now I do not think adjusting time may be necessary. I can revisit this if required.